agecheck-js.js

```diff
@@ -16,49 +14,45 @@
   var acYesBtn = document.getElementById('acYesBtn');
   var acNoBtn = document.getElementById('acNoBtn');
 
-  function isAgeCheckDisplayed() {
-    return localStorage.getItem(ageCheckDisplayedKey) != null;
-  }
+  //Create Cookie
+  function CreateCookie(name,value,days) {
+    var date= new Date();
+    date.setDate(date.getDate()+days);
+    var expires = "; expires="+date+"; path=/";
 
-  function setAgeCheckDisplayed(val) {
-    var isDisplayed = val == null ? true : val;
-    localStorage.setItem(ageCheckDisplayedKey, isDisplayed);
+    document.cookie = name+"="+value+expires;
   }
 
-  function getAgeCheckDisplayExpiryDate() {
-    var datestring = localStorage.getItem(ageCheckDisplayedExpiryDateKey);
-    return new Date(datestring);
+  //Read Cookie
+  function ReadCookie(name) {
+    var nameEquals = name + "=";
+    var ca = document.cookie.split(';');
+
+    for(var i=0; i < ca.length; i++) {
+      var c = ca[i];
+      while (c.charAt(0)==' ') c = c.substring(1,c.length);
+      if (c.indexOf(nameEquals) == 0) return c.substring(nameEquals.length,c.length);
+    }
+    return null;
   }
 
-  function getAgeCheckExpiryDate(days) {
-    var now = new Date();
-    var expiryDate = new Date();
-    expiryDate.setDate(now.getDate() + days);
-    return expiryDate;
+  function isAgeCheckDisplayed() {
+    return ReadCookie(ageCheckDisplayedKey) != null;
   }
 
-  function setAgeCheckDisplayedExpiryDate(days)  {
-    var expiryDays = days == null ? ageCheckDisplayedExpiryDays : days;
-    var expiryDate = getAgeCheckExpiryDate(expiryDays);
-    localStorage.setItem(ageCheckDisplayedExpiryDateKey, expiryDate);
+  function setAgeCheckDisplayed(val) {
+    var isDisplayed = val == null ? true : val;
+    CreateCookie(ageCheckDisplayedKey, isDisplayed, ageCheckDisplayedExpiryDays);
   }
 
+
   function setDisplayed() {
     setAgeCheckDisplayed();
-    setAgeCheckDisplayedExpiryDate();
     acModal.style.display = 'none';
   }
 
-  function hasAgeCheckDisplayExpired() {
-    var now = new Date();
-    return getAgeCheckDisplayExpiryDate() <= now;
-  }
-
   function ageCheckDisplayedCheck() {
-    acModal.style.display = !isAgeCheckDisplayed() || hasAgeCheckDisplayExpired() ? "block" : "none";
-      if(isAgeCheckDisplayed() && hasAgeCheckDisplayExpired()) {
-        setAgeCheckDisplayed(null);
-      }
+    acModal.style.display = !isAgeCheckDisplayed() ? "block" : "none";
```

Comment:
Could make this slightly simpler
`acModal.style.display = isAgeCheckDisplayed() ? "none" : "block";`